maestro/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for maestro.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: overture/maestro
+  tag: 0.1.0-SNAPSHOT
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 11235
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: maestro.local
+      paths: []
+
+  tls: []
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+extraEnv: []
+
+errorslogVolume:
+  size: 1Gi
+  storageClass: standard
+
+containerPort: 80
```

Comment:
This value is trivial

Reply:
it's ok I wanted to keep it here in case people want to change it for any reason and because it's a property that is passed to spring server.port